source/main/services/biometrics.ts

```diff
@@ -67,7 +67,11 @@ export async function sourceEnabledForBiometricUnlock(sourceID: VaultSourceID):
 }
 
 export async function supportsBiometricUnlock(): Promise<boolean> {
-    return systemPreferences.canPromptTouchID();
+    try {
+        return systemPreferences.canPromptTouchID();
+    } catch (err) {
+        return false;
+    }
```

Comment:
```suggestion
	if (process.platform === 'darwin') {
        return systemPreferences.canPromptTouchID();
    }

    return false;
```

I don't think it really matters either way 😄

Reply:
In [Electron docs](https://www.electronjs.org/docs/api/system-preferences#systempreferencescanprompttouchid-macos), it is written that `systemPreferences.canPromptTouchID()` is a MacOS-only feature. So it really don't matter ...